roles/faucet/templates/faucet.service.j2

```diff
@@ -2,10 +2,10 @@
 Description="Cosmos REST faucet"
 
 [Service]
-User={{node_user}}
-Environment="PATH={{chain_home}}/cosmovisor/current/bin"
+User={{ node_user }}
+Environment="{{ faucet_chain_env }}"
 WorkingDirectory={{node_user_home}}/cosmos-rest-faucet
-ExecStart={{node_user_home}}/cosmos-rest-faucet/.env/bin/hypercorn cosmos_rest_faucet:app -b 0.0.0.0:8000
+ExecStart={{node_user_home}}/cosmos-rest-faucet/.env/bin/hypercorn cosmos_rest_faucet:app -b 0.0.0.0:{{ faucet_port }}
```

Comment:
```suggestion
ExecStart={{node_user_home}}/cosmos-rest-faucet/.env/bin/hypercorn cosmos_rest_faucet:app -b [::]:{{ faucet_port }}
```

This will allow the service to listen on both IPv4 and IPv6.